pkg/archive/jar.go

```diff
@@ -61,3 +62,43 @@ var StableJARBuildMetadata = ZipEntryStabilizer{
 		zf.SetContent(buf.Bytes())
 	},
 }
+
+var StableJAROrderOfAttributeValues = ZipEntryStabilizer{
+	Name: "jar-attribute-value-order",
+	Func: func(zf *MutableZipFile) {
+		if !strings.HasSuffix(zf.Name, "META-INF/MANIFEST.MF") {
+			return
+		}
+		r, err := zf.Open()
+		if err != nil {
+			return
+		}
+		manifest, err := ParseManifest(r)
+		if err != nil {
+			return
+		}
+		for _, attr := range []string{
+			"Export-Package",
+			"Include-Resource",
+			"Provide-Capability",
+			"Private-Package",
+		} {
+			value, _ := manifest.MainSection.Get(attr)
+			// Skip empty values
+			if value == "" {
+				continue
+			}
+			value = strings.ReplaceAll(value, "\r", "")
```

Comment:
are you sure you need to add these replacements? I believe `Get` will return the string already cleaned of whitespace (might be wrong though)

Reply:
You are right. `normalizeLineEndings` handles that.